README.md

```diff
@@ -469,6 +472,53 @@ import { FormControl, NgValidatorsErrors } from '@ngneat/reactive-forms';
 
 const control = new FormControl<string, NgValidatorsErrors>();
 ```
+## Control Operators
+
+Each `valueChanges` or `values$` takes an operator `diff()`, which emits only changed parts of form:
+
+```ts
+import { FormControl } from '@ngneat/reactive-forms';
+
+const control = new FormControl<string>('');
+control.valueChanges
+  .pipe(diff())
+  .subscribe(value => {
+    // value is emited only if it has been changed
```

Comment:
value is emitted only if it has been changed, and only the changed parts.

Reply:
updated